app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(elves: list) -> int:
```

Comment:
You could calculate the total rating of all team players, not only elves. Change `elves` to `players` and indicate in the type annotation that you expect to obtain a list of `Player` class instances, not a plain list of any types.

Reply:
Thanks for the detailed answer regarding the type annotation in the list